acceptance/cluster/localcluster.go

```diff
@@ -580,11 +581,24 @@ func (l *LocalCluster) Start() {
 	}
 }
 
+func (l *LocalCluster) checkConsistency(t *testing.T) {
+	if l.NumNodes() <= 0 {
+		return
+	}
+	// Always connect to the first node in the cluster.
+	kvClient, kvStopper := l.NewClient(t, 0)
+	defer kvStopper.Stop()
+	if pErr := kvClient.CheckConsistency(keys.TableDataMin, keys.TableDataMax); pErr != nil {
+		t.Fatal(pErr)
+	}
+}
+
 // Assert drains the Events channel and compares the actual events with those
 // expected to have been generated by the operations performed on the nodes in
 // the cluster (restart, kill, ...). In the event of a mismatch, the passed
 // Tester receives a fatal error.
 func (l *LocalCluster) Assert(t *testing.T) {
+	l.checkConsistency(t)
```

Comment:
where I'm going with this is that this may be better kept as an extra method on Cluster than folded into Assert.